lua/oil/adapters/files.lua

```diff
@@ -377,7 +378,12 @@ M.render_action = function(action)
   elseif action.type == "delete" then
     local _, path = util.parse_url(action.url)
     assert(path)
-    return string.format("DELETE %s", M.to_short_os_path(path, action.entry_type))
+    local short_path = M.to_short_os_path(path, action.entry_type)
+    if config.delete_to_trash then
+      return string.format(" TRASH %s", short_path)
```

Comment:
Is the space there intentional?

Reply:
Yes, it's to right align all of the action verbs